src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -110,3 +115,110 @@ def create_if_dcg(
 def create_workflow_actions(if_dcg: DataConditionGroup, actions: list[dict[str, Any]]):
     # TODO: create actions, need registry
     pass
+
+
+def update_migrated_issue_alert(rule: Rule, data: dict[str, Any], user: RpcUser | None = None):
+    try:
+        alert_rule_workflow = AlertRuleWorkflow.objects.get(rule=rule)
+    except AlertRuleWorkflow.DoesNotExist:
+        logger.exception("AlertRuleWorkflow does not exist", extra={"rule_id": rule.id})
+        return
+
+    workflow: Workflow = alert_rule_workflow.workflow
+    if not workflow.when_condition_group:
+        # won't let me throw a logger exception here
```

Comment:
you can throw `logger.error`, it will complain if you throw `logger.exception` outside of a try/except